apps/custom-ui-floating-window-demo/README.md

```diff
@@ -0,0 +1,40 @@
+### Floating Window Demo App
+
+<img width="600" alt="Screenshot 2022-12-01 at 01 58 24" src="https://user-images.githubusercontent.com/19341550/204933908-4d9cdc9f-4e93-424d-ab14-c6375e6c707e.png">
+
+<img width="600" alt="Screenshot 2022-12-01 at 01 59 39" src="https://user-images.githubusercontent.com/19341550/204933903-3c3a8100-e539-46d0-a399-e271de717d0d.png">
+
+<img width="600" alt="Screenshot 2022-12-01 at 02 00 34" src="https://user-images.githubusercontent.com/19341550/204933896-d7fcd114-7f4e-43d9-846b-95dbcf92c945.png">
+
+### Running the app
+
+1. Clone the repo, `cd` into the app directory, and run `npm i` to install all the required dependencies.
+
+2. Use ngrok to expose the port 3000 using `ngrok http 3000` command. Create a Pipedrive app and add Custom UI Floating window with the ngrok URL pointing to root.
+
+3. Based on the created Pipedrive app, copy the client id, secret to the `.env.example` file. Rename this file to `.env`.
+
+4. Generate the database by using the `npx prisma db push` command.
+
+```
+% npx prisma db push
+Environment variables loaded from .env
+Prisma schema loaded from prisma/schema.prisma
+Datasource "db": SQLite database "dev.db" at "file:./dev.db"
+
+SQLite database dev.db created at file:./dev.db
+
+🚀  Your database is now in sync with your Prisma schema. Done in 28ms
+
+✔ Generated Prisma Client (4.6.1 | library) to ./node_modules/@prisma/client in 58ms
+```
+
+5. You can start the app by running the `npm run dev` command.
```

Comment:
Probably need to document here about steps to create app - which scopes to choose, how to create floating window etc.